### PR TITLE
Add Merative copyright

### PR DESCRIPTION
**Pull request by dcwebermn**

### CODE REVIEW

File: src/pages/clouddocs/annotator_attribute_detection.md [hunk 1]
```diff
@@ -5,19 +5,13 @@ categories: Annotators
 slug: annotator_attribute_detection
 toc: true
 ---
-<!-- ---
+<!--                                                                    -->
+<!-- (C) Copyright Merative US L.P. and others 2011, 2023                -->
```
Comment (cagrev) [inline src/pages/clouddocs/annotator_attribute_detection.md:9]: This one had an original date of 2015. Not sure if we want to retain that or not?

File: src/pages/clouddocs/annotator_relation.md [hunk 2]
```diff
@@ -5,19 +5,13 @@ categories: Annotators
...
-lastupdated: "2019-04-12"
-
-keywords: annotator clinical data, clinical data, annotation
-
-subcollection: wh-acd
-
----
-
-# Relation -->
+<!-- # Sections -->
```
Comment (cagrev) [inline src/pages/clouddocs/annotator_relation.md:14]: This should still say Relation, not Sections.

File: src/pages/clouddocs/annotator_smoking.md [hunk 3]
```diff
@@ -5,19 +5,13 @@ categories: Annotators
...
-lastupdated: "2019-09-21"
-
-keywords: annotator clinical data, clinical data, annotation
-
-subcollection: wh-acd
-
----
-
-# Smoking (Deprecated) -->
+<!-- # Smoking -->
```
Comment (cagrev) [inline src/pages/clouddocs/annotator_smoking.md:14]: Is this one still deprecated or not? Doesn't really matter in the comment, but helpful to keep consistent with what is presented, and that shows deprecated today.

File: src/pages/clouddocs/clinical_insights_cartridge.md [hunk 4]
```diff
@@ -5,19 +5,13 @@ categories: Clinical Insights
 slug: clinical_insights_cartridge
 toc: true
 ---
-<!-- ---
+<!--                                                                    -->
+<!-- (C) Copyright Merative US L.P. and others 2011, 2023               -->
```
Comment (cagrev) [inline src/pages/clouddocs/clinical_insights_cartridge.md:9]: Use start date of 2020 per the lines removed?

File: src/pages/clouddocs/clinical_insights_diagnosis.md [hunk 5]
```diff
@@ -5,19 +5,13 @@ categories: Clinical Insights
 slug: clinical_insights_diagnosis
 toc: true
 ---
-<!-- ---
+<!--                                                                    -->
+<!-- (C) Copyright Merative US L.P. and others 2011, 2023               -->
```
Comment (cagrev) [inline src/pages/clouddocs/clinical_insights_diagnosis.md:9]: Use start date of 2020 per the lines removed?

File: src/pages/clouddocs/clinical_insights_medication.md [hunk 6]
```diff
@@ -5,19 +5,13 @@ categories: Clinical Insights
 slug: clinical_insights_medication
 toc: true
 ---
-<!-- ---
+<!--                                                                    -->
+<!-- (C) Copyright Merative US L.P. and others 2011, 2023               -->
```
Comment (cagrev) [inline src/pages/clouddocs/clinical_insights_medication.md:9]: Start date of 2020?

File: src/pages/clouddocs/clinical_insights_normality.md [hunk 7]
```diff
@@ -5,19 +5,13 @@ categories: Clinical Insights
 slug: clinical_insights_normality
 toc: true
 ---
-<!-- ---
+<!--                                                                    -->
+<!-- (C) Copyright Merative US L.P. and others 2011, 2023               -->
```
Comment (cagrev) [inline src/pages/clouddocs/clinical_insights_normality.md:9]: Start date of 2020?

File: src/pages/migration/considerations.md [hunk 8]
```diff
@@ -5,6 +5,12 @@ categories: migration
 slug: considerations
 toc: true
 ---
+<!--                                                                    -->
+<!-- (C) Copyright Merative US L.P. and others 2018, 2023                -->
```
Comment (cagrev) [inline src/pages/migration/considerations.md:9]: Use start date of 2022.

File: src/pages/migration/migrate-file-storage.md [hunk 9]
```diff
@@ -5,6 +5,12 @@ categories: migration
 slug: migrate-file-storage
 toc: true
 ---
+<!--                                                                    -->
+<!-- (C) Copyright Merative US L.P. and others 2018, 2023                -->
```
Comment (cagrev) [inline src/pages/migration/migrate-file-storage.md:9]: Use start date of 2022.

File: src/pages/migration/migrate-object-storage.md [hunk 10]
```diff
@@ -5,6 +5,12 @@ categories: migration
 slug: migrate-object-storage
 toc: true
 ---
+<!--                                                                    -->
+<!-- (C) Copyright Merative US L.P. and others 2018, 2023                -->
```
Comment (cagrev) [inline src/pages/migration/migrate-object-storage.md:9]: Use start date of 2022.

**Review by cagrev — APPROVED** [review]
Looks good. Several comments to address with regard to the start date for the copyrights, but otherwise didn't see any major issues.

**Review by dlangst — APPROVED** [review]
LGTM

**Pull request merged**